### PR TITLE
Implement user type checking

### PR DESCRIPTION
**Pull request by maripot**
multiple models #still no home page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,11 @@
 
 ## Info
 
+*FAQ 
+
+"action controller doesn't recognize term input"
```
Comment (alexthemitchell) [inline README.md]: This isn't a question, but is located in the `FAQ` section. I'd recommend either moving it to a "troubleshooting" section or phrase it as a question.

Also, enclose the *actual command* in ticks. `db:drop db:create db:migrate` isn't a terminal program. You probably mean `rake db:drop db:create db:migrate` or something like that. Please update.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
 class ApplicationController < ActionController::Base
+  protected
+
+  def after_sign_in_path_for(resource)
+    # return the path based on resource
```
Comment (alexthemitchell) [inline app/controllers/application_controller.rb]: What does this do/do we need it?

Comment (maripot) [inline app/controllers/application_controller.rb]: i think we will need this for assigning different roles to admin/user model - it was mentioned in the tutorial as needed for that but we haven't gotten to that part yet so im not quite sure; if we end up not needing it we can get rid of it

**Comment by alexthemitchell** [comment]
Closes #45

**Comment by alexthemitchell** [comment]
Closes #44

**Pull request merged**